File: app/src/main/java/sgtmelon/scriptum/app/view/activity/PreferenceActivity.java
package sgtmelon.scriptum.app.view.activity;

import android.graphics.drawable.Drawable;
import android.os.Bundle;
import android.util.Log;

import androidx.annotation.Nullable;
import androidx.appcompat.widget.Toolbar;
import sgtmelon.scriptum.R;
import sgtmelon.scriptum.app.view.fragment.PreferenceFragment;
import sgtmelon.scriptum.app.view.parent.BaseActivityParent;
import sgtmelon.scriptum.office.utils.ColorUtils;

public final class PreferenceActivity extends BaseActivityParent {

    private static final String TAG = PreferenceActivity.class.getSimpleName();

    @Override
    protected void onCreate(@Nullable Bundle savedInstanceState) {
        Log.i(TAG, "onCreate");
        super.onCreate(savedInstanceState);
        setContentView(R.layout.activity_preference);

        setupToolbar();

        getFragmentManager()
                .beginTransaction()
                .replace(R.id.fragment_container, new PreferenceFragment())
                .commit();
    }

    private void setupToolbar() {
        Log.i(TAG, "setupToolbar");

        final Toolbar toolbar = findViewById(R.id.toolbar);
        toolbar.setTitle(getString(R.string.title_preference));

        final Drawable arrowIcon = ColorUtils.INSTANCE.getDrawable(
                this, R.drawable.ic_cancel_exit, R.attr.clContent
        );

        toolbar.setNavigationIcon(arrowIcon);
        toolbar.setNavigationOnClickListener(view -> finish());
    }

}

File: app/src/main/java/sgtmelon/scriptum/app/screen/preference/PreferenceActivity.kt
package sgtmelon.scriptum.app.screen.preference

import android.os.Bundle
import androidx.appcompat.widget.Toolbar
import sgtmelon.scriptum.R
import sgtmelon.scriptum.app.screen.BaseActivityParent
import sgtmelon.scriptum.office.utils.ColorUtils.getDrawable

class PreferenceActivity : BaseActivityParent() {

    override fun onCreate(savedInstanceState: Bundle?) {
        super.onCreate(savedInstanceState)
        setContentView(R.layout.activity_preference)

        setupToolbar()

        // TODO исправить
        fragmentManager
                .beginTransaction()
                .replace(R.id.preference_fragment_container, PreferenceFragment())
                .commit()
    }

    private fun setupToolbar() {
        val toolbar: Toolbar = findViewById(R.id.toolbar_container)

        toolbar.title = getString(R.string.title_preference)
        toolbar.navigationIcon = getDrawable(R.drawable.ic_cancel_exit, R.attr.clContent)
        toolbar.setNavigationOnClickListener { finish() }
    }

}